vrk-kpa/suomidigi
# SUOPA-794 hide menuitems

There are "first-level" menuitems that should be hidden on the desktop view such as "Tietoa Suomidigistä", that should have sidebar menu enabled but first level disabled on the main menu.

Testing Guide:
- make fresh
- Create First level menu item which has at least one second level item.
- Add is-hidden--desktop class to Container Class in Attributes (or Omainaisuudet)
![image](https://user-images.githubusercontent.com/30072786/72719362-5d932b00-3b80-11ea-8f86-fe1f8ca73b5e.png)

Check:
- First level item should not be displayed on Desktop-menu, but should be displayed on mobile menu.
- When on that First level menu item page, the sidebar menu should be displayed normally.
